lua/languages/global/utils/init.lua

```diff
@@ -1,7 +1,20 @@
+local lsp_installer_servers = require("nvim-lsp-installer.servers")
 local global = require("core.global")
 
 local M = {}
 
+M.setup_lsp = function (server_name, start_fn)
+    local ok , server = lsp_installer_servers.get_server(server_name)
+    if ok then
+        server:on_ready(start_fn)
+        if not server:is_installed() then
+            server:install()
+        end
+    else
+        -- TODO: print error
+    end
+end
```

Comment:
Adding a utility function that would help reduce boilerplate-

Also, as long as you provide a valid `server_name`, this should never happen (unsurprisingly). Figured maybe an error message could be nice here.